File: 2_web_development_101/euler2.js
var fib = function(n){
  if (n == 1 || n == 2) {
    return 1;
  }
  else {
    return fib(n-1)+fib(n-2);
  }
};
var y = 1;
var x = 0;
while (x < 4000000) {
  var z = fib(y);
  if (z % 2 == 0) {
    x = x + z;
  }
  y = y + 1;
  if (x > 4000000) {
    x = x - z;
    break;
  }
}
console.log(x)

File: 2_web_development_101/euler2.py
def fib(n):
    if n == 1 or n == 2:
        return 1
    else:
        return fib(n-1) + fib(n-2)
y = 1
x = 0
while x < 4000000:
    z = fib(y)
    if z % 2 == 0:
        x = x + z
    y = y + 1        
print x
